source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md

```diff
@@ -27,11 +27,11 @@ Before you replace Spark Standalone Cluster Manager with the one provided by BDP
 * Have CRDT enabled and activated as a bucket type. See how to do that [here][riak data types].
 * Have an operational Spark cluster (version 1.4.0+). 
 * Have the following information available:
-IP addresses of Riak cluster nodes and protobuf ports. They will look like: 172.31.9.125:10017,172.31.9.126:10017,172.31.9.127:10017.
-IP addresses and ports of LES service. They will look  like: 172.31.9.125:10012,172.31.9.126,172.31.9.127:10012.
-Name of the group for leader election. (It is set to 'spark_leaders' by default.)
-Name of consistent bucket for Spark metadata storage (you set this up during [configuration][bdp configure]).
-Name of CRDT map for metadata storage (i.e. 'spark-cluster-map'). The CRDT map stores serialized classes containing Workers, Applications and Jobs data. This data is stored in the map as a key-value pair with key composed of a prefix of entity type plus unique id (i.e. app_42343sdfd33), and value containing serialized (by serializer passed in by spark master) class for Worker, Application or Job.
+  * IP addresses of Riak cluster nodes and protobuf ports. They will look like: `172.31.9.125:10017, 172.31.9.126:10017, 172.31.9.127:10017`.
+  * IP addresses and ports of LES service. They will look  like: `172.31.9.125:10012, 172.31.9.126:10012, 172.31.9.127:10012`.
```

Comment:
Neither. They should not be in code elements. This is simply a list of information a user needs, not specific commands to obtain it or quotes from its location within a codeblock.